addon/index.js

```diff
@@ -359,6 +361,15 @@ function installExperiment(experiment) {
   }
 }
 
+function uninstallSelf() {
+  // First, kick out all the experiment add-ons
+  Object.keys(store.installedAddons).forEach(id => {
+    uninstallExperiment({addon_id: id});
+  });
```

Comment:
Can you remove the line referencing `uninstall-all` in `addon/README.md` then I'm good to merge.